db/migrate/20200614123306_add_support_webmentions_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class AddSupportWebmentionsToArticles < ActiveRecord::Migration[6.0]
+  def change
+    add_column :articles, :support_webmentions, :boolean, default: false
```

Comment:
I'm not saying this can go here, but right now the articles table has 76 columns (according to `article.attributes.keys.size`) and I think we should weigh carefully wether or not we add another one especially given that (a) it will conceivably be false for a majority of the articles (at least until webmentions gain wider support) and (b) is only needed in relatively specific circumstances (article creation and updates) where maybe we can live with an extra query here or there, especially if it's made by an async worker.

Reply:
@citizen428 I understand your concern and would like to explore a different approach if you could point me in a different direction! 

Is it possible to persist a virtual attribute until a worker has finished running 🤔

Reply:
If it's reasonably shortlived, Redis seems like a good choice.

Reply:
You can use Redis. Add the attribute, then remove it at the end. It might not be needed as mentioned in the other reviews